main.tf

```diff
@@ -278,6 +278,11 @@ module "accounting_svc" {
   root_path = var.accounting_base_path
 }
 
+module "billing_cost_management" {
+  source     = "./billing_cost_management"
+  aws_region = local.aws_region
```

Comment:
Since there is no `aws_region` module variable, you need to remove this line

Reply:
Thanks!